src/elements/ChangeOwnerPopUp/ChangeOwnerPopUp.js

```diff
@@ -34,7 +34,7 @@ import { ReactComponent as SearchIcon } from 'igz-controls/images/search.svg'
 
 import './changeOwnerPopUp.scss'
 
-const ChangeOwnerPopUp = ({ changeOwnerCallback, projectId, setNotification }) => {
+const ChangeOwnerPopUp = ({ changeOwnerCallback, dispatch, projectId, setNotification }) => {
```

Comment:
You don't need to pass `setNotification` into the component. Instead just use an import